README.md

```diff
@@ -5,4 +5,13 @@ easy-schema-examples
 A collection of examples of the XML schema's used by EASY. These schema's are published at 
 [https://easy.dans.knaw.nl/schemas/](https://easy.dans.knaw.nl/schemas/)
 
+Development cycle
```

Comment:
a couple of thoughts about this section:

1. Great to see such a detailed description! It makes the development cycle very clear.
2. Rather than `mvncist`, use `mvn clean generate-test-resources` (bullet 3).
3. Should there be a mention here of when to set `easy-schema` version numbers in `pom.xml` for `easy.schema.version` to certain values?
4. Regarding the last bullet, saying "_The test ignored above should now succeed._" - is this guaranteed? Are there situations where this test can still fail?

Reply:
@3 Is this a question for @lindareijnhoudt 

@4 @lindareijnhoudt requested a test like this.